spec/avram/operations/save_operation_spec.cr

```diff
@@ -67,6 +67,7 @@ end
 
 private class UpsertUserOperation < User::SaveOperation
   upsert_lookup_columns :name, :nickname
+  upsert_unique_on :name, :nickname
```

Comment:
🤔 why both?

Reply:
Different APIs - one indicates a unique constraint, the other one is used for the find or create by and doesn’t have any requirement for mapping to UKs.

Reply:
They just happen to be the same, at least until we decide on which database migration is best to add for any unique keys.